Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -440,6 +442,144 @@ theorem lift_tprod : lift (tprod R : MultilinearMap R s _) = LinearMap.id :=
   Eq.symm <| lift.unique' rfl
 #align pi_tensor_product.lift_tprod PiTensorProduct.lift_tprod
 
+end lift
+
+section map
+
+variable {t t' : ι → Type*}
+variable [∀ i, AddCommMonoid (t i)] [∀ i, Module R (t i)]
+variable [∀ i, AddCommMonoid (t' i)] [∀ i, Module R (t' i)]
+
+variable (R s t) in
+
+/--
+Let `sᵢ` and `tᵢ` be family of `R`-modules, a map from `Π i, sᵢ → tᵢ` to multilinear maps
+`(Π i, sᵢ) → ⨂ tᵢ` by `(fᵢ) ↦ v ↦ ⨂ₜ fᵢ vᵢ`. This map is multilinear as well.
+-/
+@[simps]
+def _root_.MultilinearMap.piLinearMapToPiTensorProduct :
+    MultilinearMap R (fun i ↦ s i →ₗ[R] t i) (MultilinearMap R s (⨂[R] (i : ι), t i)) :=
+{ toFun := fun f ↦
+  { toFun := fun v ↦ tprod R fun j ↦ (f j) (v j)
+    map_add' := fun v i a b ↦ by
+      dsimp
+      simp_rw [show ∀ (j : ι) (x : s i), (f j) (update v i x j) =
+        update (fun k ↦ (f k) (v k)) i (f _ x) j by
+        · intro j x
+          by_cases h : j = i
+          · subst h; simp only [update_same]
+          · rw [update_noteq h, update_noteq h], map_add, MultilinearMap.map_add]
+    map_smul' := fun v i r x ↦ by
+      dsimp
+      simp_rw [show ∀ (j : ι) (x : s i), (f j) (update v i (r • x) j) =
+        update (fun k ↦ (f k) (v k)) i (f _ (r • x)) j by
+      · intro j x
+        by_cases h : j = i
+        · subst h; simp only [update_same]
+        · rw [update_noteq h, update_noteq h], map_smul, MultilinearMap.map_smul]
+      congr
+      ext j
+      by_cases h : j = i
+      · subst h; simp only [update_same]
+      · rw [update_noteq h, update_noteq h] }
```

Comment:
```suggestion
{ toFun := (tprod R).compLinearMap
```

Reply:
There is `MultilinearMap.compLinearMapMultilinear`, but I am not sure this is what we want here

Reply:
It's `(LinearMap.applyₗ (tprod R)).compMultilinearMap compLinearMapMultilinear`

Reply:
Thanks!